diff --git a/wrangler-dist/cli.js b/wrangler-dist/cli.js
index 843a7026fbc56d9b7f84442cc24773b486ff99f8..06a438c4676f56e6dab98139c112cf71099a3591 100644
--- a/wrangler-dist/cli.js
+++ b/wrangler-dist/cli.js
@@ -170331,7 +170331,7 @@ You can also run \`wrangler whoami\` to check your current authentication status
       }
       handleErrorEvent(event) {
         if (event.cause instanceof miniflare.MiniflareCoreError && event.cause.isUserError()) this.emit("error", new UserError(event.cause.message, { telemetryMessage: "api dev miniflare user error" }));
-        else if (event.source === "ProxyController" && event.reason.startsWith("Failed to send message to")) {
+        else if (event.source === "ProxyController" && (event.reason.startsWith("Failed to send message to") || event.reason.startsWith("Error inside ProxyWorker"))) {
           logger3.debug(`Error in ${event.source}: ${event.reason}
 `, event.cause);
           logger3.debug("=> Error contextual data:", event.data);
@@ -354369,7 +354369,7 @@ var init_DevEnv = __esm({
               telemetryMessage: "api dev miniflare user error"
             })
           );
-        } else if (event.source === "ProxyController" && (event.reason.startsWith("Failed to send message to") || event.reason.startsWith("Could not connect to InspectorProxyWorker"))) {
+        } else if (event.source === "ProxyController" && (event.reason.startsWith("Failed to send message to") || event.reason.startsWith("Could not connect to InspectorProxyWorker") || event.reason.startsWith("Error inside ProxyWorker"))) {
           logger2.debug(`Error in ${event.source}: ${event.reason}
 `, event.cause);
           logger2.debug("=> Error contextual data:", event.data);
